censys_connector.py

```diff
@@ -278,7 +276,9 @@ def _handle_search(
 
             results = response_json.get("results", [])
             if (
-                limit and page == num_pages and (limit % CENSYS_QUERY_CERTIFICATE_DATA_PER_PAGE) != 0
+                limit
```

Comment:
@mposluszny-splunk, We are following a practice of fixing these W503 flake8 warnings in the code. Reference: https://github.com/splunk-soar-connectors/threatstream/blob/next/threatstream_connector.py#L4045

Reply:
The way this has been done by Michał is IMHO more readable and in line with a tool like black (which is a de facto standard or is becoming one). How and when has this been agreed upon, can we further discuss it?

Reply:
W503 warning was addressed by me in review in appscript repo. It has incorrect flags usage regarding flake8. We should extend, not override base flags. Without this change it will throw either W503 or W504 as these one are conflicting

Reply:
For now changes for this line will be reverted, together with disabling black formatting. Those changes will be applied after resolving issues with current platform compile script